src/presentation/ui/page/runtimes.templ

```diff
@@ -0,0 +1,98 @@
+package page
+
+import (
+	"github.com/speedianet/os/src/domain/valueObject"
+	componentRuntimes "github.com/speedianet/os/src/presentation/ui/component/runtimes"
+	componentStructural "github.com/speedianet/os/src/presentation/ui/component/structural"
+	presenterDto "github.com/speedianet/os/src/presentation/ui/presenter/dto"
+)
+
+script RuntimesIndexLocalState(defaultVhostHostname, defaultRuntimeType string) {
+    document.addEventListener('alpine:init', () => {
+		Alpine.data('runtimes', () => ({
+			reloadPage(vhostHostname, runtimeType) {
+				htmx.ajax(
+					'GET',
+					'/runtimes/?vhostHostname=' + vhostHostname + '&runtimeType=' + runtimeType,
+					{
+						select: '#runtimes-page-content',
+						target: '#runtimes-page-content',
+						indicator: '#loading-overlay',
+						swap: 'outerHTML transition:true'
+					},
+				);
+			},
+			changeSelectedVhostHostname(selectedVhostHostname) {
+				this.reloadPage(selectedVhostHostname, defaultRuntimeType);
+			},
```

Comment:
The reason behind creating the same request for Alpine.js to execute is because it is using states. I'll try to explain the nuances of this type of page to better illustrate why I found the need for this method.

There are two values that are parameters for my page request: the selected `vhostHostname` and the selected `runtimeType`. The `vhostHostname` is changed by a `<select>`, while the `runtimeType` is changed by a `<button>` from the `HorizontalTabHeader`. In other words, we're talking about two components using Alpine.js to modify the value of something, as they react to `onClick`/`onChange` events.

Knowing this, the `vhostHostname` and `runtimeType` values coming from Go are immutable. Therefore, having HTMX make the request simply by listening to a value change event won't be enough. I would need to inject the Alpine.js state into the route as a route parameter, just like it's done with the values from Go. This would be disastrous. That's exactly why I had to replicate the request, but this time using the values from Alpine.js, because this request is using the values injected by Alpine.js, not those from Go.